app/src/main/java/bean/pwr/imskamieskiego/MapActivity.java

```diff
@@ -27,7 +33,16 @@
     private FloatingActionButton patientAssistantButton;
     private FloatingActionButton foodButton;
     private FloatingActionButton quickAccessButton;
+    private Button patientAssistantButtonDescription;
+    private Button foodButtonDescription;
+    private Button wcButtonDescription;
     private ImageButton changeFloorButton;
+    private ConstraintLayout layoutBottomSheet;
+    private Button guideToButton;
+    private TextView placeName;
+    private BottomSheetBehavior sheetBehavior;
```

Comment:
BootomSheet wcale nie musi być na dole, jak wskazuje na to nazwa. Myślę, że lepszą nazwą dla tego komponentu było np. infoSheet. Ogólnie zmienić na nazwę która wskazuje na funkcję (bo to jest najważniejsze), a nie na położenie (bo to może się w każdej chwili zmienić)